src/components/entry/EntryDetails.js

```diff
@@ -0,0 +1,18 @@
+import { useRouteMatch } from 'react-router-dom'
+
+const EntryDetails = () => {
+  const { id } = useRouteMatch().params
+
+  return (
+    <div>
+      <br />
+      <br />
+      <br />
+      <br />
+      <br />
+      <p>EntryDetails for id: {id}</p>
```

Comment:
beautiful, absolutely ground-breaking code 😍

Reply:
Glad you noticed 😝